prov/efa/src/efa_hmem.c

```diff
@@ -94,7 +94,9 @@ static int efa_hmem_info_init_protocol_thresholds(struct efa_hmem_info *info, en
 static int efa_hmem_info_init_system(struct efa_hmem_info *system_info, struct efa_domain *efa_domain)
 {
 	system_info->initialized = true;
+	system_info->p2p_required = false;
```

Comment:
Changed this to `false` in the latest commit based on the fact `FI_HMEM_P2P_DISABLED` was the default `ep->hmem_p2p_opt` value when no other `FI_HMEM` interface was initialized.